src/main/resources/db/changesets/20210517_azurebillinginformation.yaml

```diff
@@ -0,0 +1,23 @@
+databaseChangeLog:
+  - changeSet:
+      id: azure_billing_information
+      author: tl
+      changes:
+        - addColumn:
+            tableName: billing_profile
+            columns:
+              - column:
+                  name: cloud_platform
+                  type: ${identifier_type}
+                  defaultValue: "GCP"
+                  constraints:
+                    nullable: false
+              - column:
+                  name: tenant_id
+                  type: ${uuid_type}
+              - column:
+                  name: subscription_id
+                  type: ${uuid_type}
+              - column:
+                  name: resource_group_id
+                  type: ${uuid_type}
```

Comment:
I still think we want this to be a resource group name

Reply:
Oh! I thought they were all uuids